package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "watch": "node scripts/watch.js index.d.ts",
+    "watch": "onchange **/*.d.ts -e out/**  -- npm run build",
```

Comment:
Yes. But don't quote after `--`, that'll make the command a single string - don't want that. We just want to not apply bash expansion on the globs.